documents/solution/micro/mse-end-to-end-canary-release-for-ack-application.yml

```diff
@@ -67,11 +67,17 @@ Parameters:
       zh-cn: 注册配置中心实例名称
     Default: test
 Resources:
+  EnableAck:
+    Type: ALIYUN::ROS::AutoEnableService
```

Comment:
It is recommended to use this module: MODULE::ACS::CS::Provision.

Reply:
Thank you for the suggestion, it has been adjusted.